core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -995,7 +995,9 @@ public boolean isRunning(String cacheName) {
       }
       CompletableFuture<Cache<?, ?>> cacheFuture = caches.get(cacheName);
       boolean started = cacheFuture != null && cacheFuture.isDone() && !cacheFuture.isCompletedExceptionally();
-      return started && cacheFuture.join().getStatus() == ComponentStatus.RUNNING;
+      if (!started) return false;
+      ComponentStatus cs = cacheFuture.join().getStatus();
+      return cs == ComponentStatus.RUNNING || cs == ComponentStatus.INITIALIZING;
```

Comment:
I'll revert this and open another JIRA to follow-up. The problem here is that when getting the cache for the REST/CLI API it verifies if the cache is running before applying an operation. Since in our case the cache is still initializing, causes an exception, returning `Unexpected error retrieving data. "ISPN012010: Cache with name 'manual' not found amongst the configured caches"` to the client, which is odd, since we just saw the cache in a list.

Reply:
Created https://issues.redhat.com/browse/ISPN-14414 to handle this.